easybuild/easyblocks/t/trilinos.py

```diff
@@ -238,8 +239,14 @@ def sanity_check_step(self):
             libs.remove('Kokkos')
             libs.append('kokkoscore')
 
+        # Get the library extension
+        if self.cfg['shared_libs']:
+            lib_ext = "." + get_shared_lib_ext()
+        else:
+            lib_ext = ".a"
```

Comment:
A one-liner here could be:

```python
lib_ext = ('a', get_shared_lib_ext())[self.cfg['shared_libs']]
```

But that's probably a bit hard to read. ;-)

I'd exclude the `.` here, that makes the below easier to read:

```python
if self.cfg['shared_libs']:
    lib_ext = get_shared_lib_ext()
else:
    lib_ext = 'a'